docs/docs-content/vm-management/architecture.md

```diff
@@ -0,0 +1,87 @@
+---
+sidebar_label: "Architecture"
+title: "Architecture"
+description: "Learn about Palette VMO pack and the architecture behind it."
+hide_table_of_contents: false
+sidebar_position: 0
+tags: ["vmo", "architecture"]
+---
+
+The Palette Virtual Machine Orchestrator (VMO) pack consolidates all components that you need to deploy and manage
+Virtual Machines (VMs) alongside containers in a Kubernetes host cluster. You can deploy VMO as an
+[add-on cluster profile](../profiles/cluster-profiles/create-cluster-profiles/create-addon-profile/create-addon-profile.md)
+on top of an existing data center or edge cluster.
+
+![Diagram that explains the architecture behind Palette VMO.](/vm-management_architecture_vmo-architecture.webp)
```

Comment:
Lovely diagram ❤️

Reply:
I do have a question about the diagram.  I see an arrow from what looks as the Palette Management Plane/Platform towards the agent.  What does that arrow represent? A communication channel?  Is it also not bi-directional? HTTPS?




![CleanShot 2024-07-22 at 08 21 45](https://github.com/user-attachments/assets/50c71ad8-76af-4e87-a0f1-af93979509cf)

Reply:
I believe Yuliia adapted this diagram from Kevin's diagram in that reference architecture document. But yes i agree that the arrow should be bidiretional. I can make the change,